dtests/src/test/scala/io/snappydata/hydra/SnappyTestUtils.scala

```diff
@@ -94,6 +94,38 @@ object SnappyTestUtils {
     return tempDir.getAbsolutePath
   }
 
+  /*
+ In case of round-off, there is a difference of .1 in snappy and spark results. We can ignore
+ such differences
+ */
+  def isIgnorable(actualRow: String, expectedRow: String): Boolean = {
+    var isIgnorable = false
+    if (actualRow != null && actualRow.size > 0 && expectedRow != null && expectedRow.size > 0) {
+      val actualArray = actualRow.split(",")
+      val expectedArray = expectedRow.split(",")
+      var diff: Double = 0.0
+      if (actualArray.length == expectedArray.length) {
+        for (i <- 0 until actualArray.length) {
+          val value1: String = actualArray(i)
+          val value2: String = expectedArray(i)
+          if (!value1.equals(value2)) {
+            try {
+              val val1: Double = value1.toDouble
+              val val2: Double = value2.toDouble
+              if (val1 > val2) diff = val1.-(val2).doubleValue
+              else diff = val2.-(val1).doubleValue
+              println("diff is " + diff)
+              if (diff <= 0.01) isIgnorable = true
```

Comment:
Add the diff <=0.1 for ignore condition instead of 0.01. The round function used in snappy tests results in the difference of 0.1 for most of the queries with the double data type.